extensions/amp-sticky-ad/1.0/amp-sticky-ad.js

```diff
@@ -193,10 +193,13 @@ class AmpStickyAd extends AMP.BaseElement {
     // `render-start` is expected to arrive first, but it's not emitted by
     // all types of ads.
     const signals = ad.signals();
-    return Promise.race([
-      signals.whenSignal(CommonSignals.RENDER_START),
-      signals.whenSignal(CommonSignals.LOAD_END),
-    ]).then(() => {
+    return this.ad_.getImpl().then(impl => {
+      const promises = [signals.whenSignal(CommonSignals.RENDER_START)];
+      if (!(impl.config || {}).renderStartImplemented) {
```

Comment:
Seems odd that this ever used load end as a signal.  I would instead of expected render start signal to be sufficient and that the AMP runtime just controls when the signal fires based on the impl config.  In other words, for DF config w/o renderStartImplemented, render start is fired when the iframe is attached.  While for configs w/ renderStartImplemented, it would be fired based on received postMessage triggered by context.renderStarted().  Are we positive that's no the case?  

Was a goal here that the sticky shouldn't be made visible until the creative had completely loaded in order to reduce blank rectangle time?